### PR TITLE
AZ-408: test_update.py returns non-zero when update failed

### PR DESCRIPTION
**Pull request by h4nsu**
This is needed so that an automatic pipeline can detect that something went wrong with the update test

### CODE REVIEW

File: local-tests/test_update.py [hunk 1]
```diff
@@ -20,10 +21,16 @@
 
 def query_runtime_version(nodes):
     print('Current version:')
+    versions = []
```
Comment (kostekIV) [inline local-tests/test_update.py:24]: wdyt about making it set already instead of a list?

File: local-tests/test_update.py [hunk 2]
```diff
@@ -20,10 +21,16 @@
...
     for i, node in enumerate(nodes):
-        sys = node.rpc('system_version').result
+        sysver = node.rpc('system_version').result
         rt = node.rpc('state_getRuntimeVersion').result['specVersion']
-        print(f'  Node {i}: system: {sys}  runtime: {rt}')
+        print(f'  Node {i}: system: {sysver}  runtime: {rt}')
+        versions.append(rt)
+    versions = set(versions)
+    if len(versions) != 1:
+        print('ERROR: nodes reported different runtime versions')
```
Comment (kostekIV) [inline local-tests/test_update.py:32]: Add reported versions to the error msg

Comment (kostekIV) [inline local-tests/test_update.py:32]: this actually won't trigger any errors in the pipeline (only an error message will be displayed). Consider changing return type to set (or dict (node -> version)) and use that information for for example trigger to `sys.exit(1)` later in the script.

File: local-tests/test_update.py (hunk 1 above)
Comment (h4nsu) [reply local-tests/test_update.py:24]: Of course, why haven't I thought about that ^^'

File: local-tests/test_update.py (hunk 2 above)
Comment (kostekIV) [reply local-tests/test_update.py:32]: It's always good to be over paranoid :) I am fine with this explanation, maybe add this "In a very unlikely ... the last query." as a comment here.

Comment (h4nsu) [reply local-tests/test_update.py:32]: Done

Comment (h4nsu) [reply local-tests/test_update.py]: Added

File: local-tests/test_update.ipynb [hunk 3]
```diff
@@ -207,6 +211,17 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9ede429e-fca2-41fe-80c7-ac7df1606ddc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if oldver == newver:\n",
+    "    print(\"ERROR: runtime version reported by nodes didn't change after the update\")"
+   ]
```
Comment (pmikolajczyk41) [inline local-tests/test_update.ipynb:224]: missing `sys.exit(1)`; also in the previous lines (~61 the notebook has changes that are not included in the script itself)

File: docker-runtime-hook/entrypoint.sh [hunk 4]
```diff
@@ -1,11 +1,10 @@
 #/bin/bash
```
Comment (pmikolajczyk41) [inline docker-runtime-hook/entrypoint.sh:1]: are these changes relevant?

**Review by pmikolajczyk41 — APPROVED** [review]
Thank you for spotting the problem and fixing it!

File: local-tests/test_update.ipynb (hunk 3 above)
Comment (h4nsu) [reply local-tests/test_update.ipynb:224]: Yes, on purpose. Notebook is not supposed to do things like sleep, sys.exit etc. It's for humans to use :)

File: docker-runtime-hook/entrypoint.sh (hunk 4 above)
Comment (h4nsu) [reply docker-runtime-hook/entrypoint.sh:1]: It's a cosmetic change that somehow fell through the cracks. As I am the original author of this file and feel a bit guilty about that, I wanted to fix it. Can I, without a separate PR, pleeeeaaaase? :P

**Review by kostekIV — APPROVED** [review]

**Pull request merged**